app/views/entries/_form.html.erb

```diff
@@ -27,22 +27,26 @@
   </div>
   <div class="form-group">
     <div class="col-sm-6">
+      <label>Work Performed</label>
```

Comment:
You have a label, it's the next line. Just remove `.sr-only` if you want it to be visible.